packages/server/src/api/controllers/integration.ts

```diff
@@ -1,7 +1,10 @@
 import { getDefinition, getDefinitions } from "../../integrations"
 import { SourceName, UserCtx } from "@budibase/types"
 
-const DISABLED_EXTERNAL_INTEGRATIONS = [SourceName.AIRTABLE]
+const DISABLED_EXTERNAL_INTEGRATIONS = [
+  SourceName.AIRTABLE,
+  SourceName.BUDIBASE,
```

Comment:
How come this has been added?

Reply:
The Budibase source wasn't in the source name list, technically its a valid source name, but don't want that key ever being returned from the integrations API.